backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/handlingunits/rest_api/HandlingUnitsRestController.java

```diff
@@ -319,21 +305,30 @@ public ResponseEntity<JsonAllowedHUClearanceStatuses> getAllowedClearanceStatuse
 	}
 
 	@PostMapping("/move")
-	public ResponseEntity<JsonGetSingleHUResponse> moveHU(
+	public void moveHU(
```

Comment:
why returning nothing after move?
what's the frontend expected to do (i.e. what will display the user) after given HU was moved?

Reply:
they are redirected to the scanner, I can't remeber why it was done like that :-?? I can dig deeper though